content/4_Gold/Stacks.problems.json

```diff
@@ -58,10 +58,9 @@
       "source": "Gold",
       "difficulty": "Normal",
       "isStarred": false,
-      "tags": ["Stack"],
+      "tags": ["Stack, Binary Search"],
```

Comment:
you did the tags wrong

Reply:
que?

Reply:
it's supposed to be 2 strings, you put them as 1

Reply:
bruh u cant tell on the website but ok